connection.js

```diff
@@ -459,6 +459,65 @@ class Connection {
         }
         /******** end of BOARDSIZES *********/
 
+        if (notification.handicap === -1 && config.noautohandicap) {
+            conn_log("no autohandicap");
+            return { reject: true, msg: "For easier bot management, automatic handicap is disabled on this bot, please manually select the number of handicap stones you want in -custom handicap-, for example 2 handicap stones" };
```

Comment:
This message implies handicap is allowed, but I suppose that's not guaranteed at this point?

So maybe just note that automatic handicap is disabled?

Reply:
i see, then we can make it smarter : 

first we shorten the first message, then we add : 

if (config.maxhandicap === 0 && notification.handicap >= 1) "this bot does not play handicap games, please set handicap to -none-(0 stones)"

if config.minhandicap >= 1 && notification.handicap === 0) "this bot plays only handicap games, please manually select the number of handicap stones you want in -custom handicap-, for example 2 handicap stones"

how about it @windo ? (would be done in a separate PR)

Reply:
update : i just realized we used the same "how about it" to end our message, when reading your 2nd message later :smiley: